### DISCUSSION TITLE
每天演好一个情绪稳定的成年人 - 好记性不如烂笔头

### DISCUSSION THREAD

**Discussion by giscus** [xiaodongxier/wiki#11] [Announcements]
# 每天演好一个情绪稳定的成年人 - 好记性不如烂笔头

王永杰的个人网站,王永杰的个人wiki,小东西儿的个人网站,xiaodongxier.com,wangyongjie.cn,前端知识管理,个人wiki,个人知识管理

https://wiki.wangyongjie.cn/%E4%B9%A6%E7%B1%8D/%E6%AF%8F%E5%A4%A9%E6%BC%94%E5%A5%BD%E4%B8%80%E4%B8%AA%E6%83%85%E7%BB%AA%E7%A8%B3%E5%AE%9A%E7%9A%84%E6%88%90%E5%B9%B4%E4%BA%BA/

**Comment by xiaodongxier** [comment]
#牛马系列 #每天演好一个情绪稳定的成年人
小时候，哭是搞定问题的绝招;
长大后，笑是面对现实的武器
